examples/AlwaysOffTraceExample.php

```diff
@@ -1,14 +1,15 @@
 <?php
 require __DIR__ . '/../vendor/autoload.php';
 
-use OpenTelemetry\Tracing\SpanContext;
-use OpenTelemetry\Tracing\Sampler\AlwaysOffSampler;
-use OpenTelemetry\Tracing\Tracer;
+use OpenTelemetry\Context\SpanContext;
+use OpenTelemetry\Trace\Sampler\AlwaysOffSampler;
+use OpenTelemetry\Trace\Tracer;
+use OpenTelemetry\Trace\TracerFactory;
 
 $sampler = AlwaysOffSampler::shouldSample();
 if ($sampler) {
-    $spanContext = SpanContext::generate(); // or extract from headers
-    $tracer = new Tracer($spanContext);
+    $tracer = TracerFactory::getInstance()
+        ->getTracer('io.opentelemetry.contrib.php');
```

Comment:
Did this naming scheme come from somewhere in particular?  I didn't see it in the spec, but I could just be missing it.

Reply:
Yes, I found something similar in the specs: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#obtaining-a-tracer

Reply:
Good enough for me!